save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/controller/AgentsController.kt

```diff
@@ -42,13 +42,18 @@ class AgentsController(
      *
      * @param execution
      * @param testSuiteDtos test suites, selected by user
+     * @param execCmd execCmd for SAVE-cli for testing in standard mode
+     * @param batchSizeForAnalyzer batchSize for SAVE-cli for testing in standard mode
      * @return OK if everything went fine.
      * @throws ResponseStatusException
      */
     @Suppress("TOO_LONG_FUNCTION", "LongMethod", "UnsafeCallOnNullableType")
     @PostMapping("/initializeAgents")
     fun initialize(@RequestPart(required = true) execution: Execution,
-                   @RequestPart(required = false) testSuiteDtos: List<TestSuiteDto>?): Mono<BodilessResponseEntity> {
+                   @RequestPart(required = false) testSuiteDtos: List<TestSuiteDto>?,
+                   @RequestPart(required = false) execCmd: String?,
```

Comment:
Aren't these available in `execution` already?

Reply:
Yes, thanks, old PR